modules/pel/tool/user_data.py

```diff
@@ -0,0 +1,112 @@
+from pel.datastream import DataStream
+from collections import OrderedDict
+from pel_values import *
+from enum import Enum, unique
+import json
+
+
+@unique
+class UserDataFormat(Enum):
+    json = 0x1
+    cbor = 0x2
+    text = 0x3
+    custom = 0x4
+
+
+class UserData:
+    """
+    This represents the User Data section in a PEL.  It is free form data
+    that the creator knows the contents of.  The component ID, version,
+    and sub-type fields in the section header are used to identify the
+    format.
+
+    The Section base class handles the section header structure that every
+    PEL section has at offset zero.
+    """
+
+    def __init__(self, stream: DataStream, sectionID: int, sectionLen: int,
+                 versionID: int, subType: int, componentID: int, creatorID: str):
+        self.stream = stream
+        self.sectionID = sectionID
+        self.sectionLen = sectionLen
+        self.versionID = versionID
+        self.subType = subType
+        self.componentID = componentID
+        self.creatorID = creatorID
+        self.dataLength = sectionLen - 8
+        self.data = self.stream.get_mem(self.dataLength)
+
+    def paser(self) -> (bool, str):
+        if self.creatorID == "B":
+            if self.componentID == 0x100:
+                from udparsers.b0100.b0100 import parseUDToJson
```

Comment:
python has an importlib module where you can import modules by name.

Reply:
> python has an importlib module where you can import modules by name.

Done, Thanks for your advice